tutorials/install.md

```diff
@@ -85,6 +124,44 @@ Build and install as follows:
     make -j4
     sudo make install
 
+#### Windows
+
+Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt for VS 2019" in the Windows search near the Start button).
+Right-click and run as Administrator.
```

Comment:
I don't think this needs to be run as admin, everything worked for me without it.